### PR TITLE
DP-29542 Avoid redirect to install.php when DB is down

### PR DESCRIPTION
**Pull request by weitzman**
**Description:**
Also see new contib module I wrote for this need - https://www.drupal.org/project/prod_no_redirect

In addition to enabling that module, this PR edits our front controller because thats the only way to override DrupalKernel.

**Jira:** (Skip unless you are MA staff)
DP-

**To Test:**
- [ ] Deploy this PR to stage or another Acquia env and performa DB restore. See if we get 500+ status code instead of the 302 redirect. The actual PHP error is not important.

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by tom-fleming** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Add a changelog YAML file to this PR</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f50c7bbd8e21e2aff41636795a5af049b17da87b
</p>

File: docroot/index.php [hunk 1]
```diff
@@ -9,11 +9,12 @@
  */
 
 use Drupal\Core\DrupalKernel;
+use Drupal\prod_no_redirect\ProdNoRedirectDrupalKernel;
 use Symfony\Component\HttpFoundation\Request;
 
 $autoloader = require_once 'autoload.php';
 
-$kernel = new DrupalKernel('prod', $autoloader);
+$kernel = new ProdNoRedirectDrupalKernel('prod', $autoloader);
```
Comment (tom-fleming) [inline docroot/index.php:17]: @weitzman When I run composer install locally, this change gets overwritten and reverted back to using `DrupalKernel`

Comment (weitzman) [reply docroot/index.php:17]: I added an instruction for drupal-scaffold composer plugin so not overwrite.

**Comment by tom-fleming** [comment]
LGTM

**Pull request merged**